packages/job-worker/src/ingest/generation.ts

```diff
@@ -565,11 +558,26 @@ export async function getRundownFromIngestData(
 		context.getStudioBlueprintConfig(),
 		showStyle.compound,
 		context.getShowStyleBlueprintConfig(showStyle.compound),
-		rundownBaselinePackages
+		rundownBaselinePackages,
+		async () => {
+			// Note: This can cause a mild race-condition, in the case of two Rundowns being created at the same time.
+			// But we're just ignoreing that for now.
+			return context.directCollections.RundownPlaylists.findFetch({
+				studioId: context.studioId,
+			})
+		},
+		async () => {
+			return context.directCollections.Rundowns.findFetch({
```

Comment:
It would be good to reduce the number of properties loaded here. The metaData property can be quite heavy, and only the playlistId looks to be needed